scipy/fft/_fftpack/__init__.py

```diff
@@ -0,0 +1,109 @@
+"""
+==================================================
+Discrete Fourier transforms (:mod:`scipy.fftpack`)
+==================================================
```

Comment:
This is a `_fftpack` module you do not need all of this docstring.

Can you remind me why it helps to move all the files from `scipy/fftpack` to `scipy/fft/_fftpack/`? It does not seem like it should be necessary. Even thinking of the upcoming backend stuff it seems like it should not matter where these (legacy) fftpack backend files live.

Reply:
Sorry, I missed the second part of your comment here. I mentioned the reason before in #10222.

If the intention is to move `scipy.fftpack` over to using pocketfft as well, then it creates a circular dependency issue.  `fft` depends on `fftpack` for the cosine and sine transforms; `fftpack` would depend on `fft` for the normal FFTs.

In the current state (where `fftpack` doesn't use pocketfft) it's not necessary and I could revert it.